libs/chatbot/src/lib/ui/chatbot.tsx

```diff
@@ -1,23 +1,217 @@
+    }),
+    actionsButtonTitle: intl.formatMessage({
+      defaultMessage: 'Actions',
+      description: 'Actions button',
+    }),
+  };
+
+  const onSubmitInputQuery = useCallback(() => {
+    const query = inputQuery.trim();
+    if (query !== '') {
+      setConversation((current) => [
+        {
+          type: ConversationItemType.Query,
+          id: getId(), // using this for now to give it a unique id, but will change later
+          date: new Date(),
+          text: query,
+        },
+        ...current,
+      ]);
+    }
+  }, [inputQuery, setConversation]);
+
+  useEffect(() => {
+    setInputQuery('');
+  }, [conversation]);
+
+  const styles = useStyles();
   return (
     <Panel
       type={panelLocation === PanelLocation.Right ? PanelType.custom : PanelType.customNear}
       isOpen={!collapsed}
-      customWidth={'300px'}
+      customWidth={'340px'}
       hasCloseButton={false}
       isBlocking={false}
-      layerProps={{ styles: { root: { zIndex: 0 } } }}
+      layerProps={{ styles: { root: { zIndex: 0, display: 'flex' } } }}
     >
-      <p>Content goes here.</p>
+      <div className={styles.container}>
+        <div className={styles.header}>
+          {/*TODO: Add icon for header*/}
+          <div className={styles.headerTitle}>{intlText.headerTitle}</div>
+          <div className={styles.headerModePill}>{intlText.pill}</div>
+          <IconButton
+            title={'Close'}
+            iconProps={{ iconName: 'Clear' }}
+            onClick={() => {
+              setCollapsed(true);
+            }}
+            className={styles.closeButton}
+          />
+        </div>
+        <div className={css(styles.content)}>
+          {conversation.map((item) => (
+            <ConversationMessage key={item.id} item={item} />
+          ))}
+        </div>
+        <div className={styles.footer}>
+          <ChatInput
+            query={inputQuery}
+            placeholder={intlText.chatInputPlaceholder}
+            isMultiline={true}
+            showCharCount={true}
+            maxQueryLength={QUERY_MAX_LENGTH}
+            submitButtonProps={{
+              title: intlText.submitButtonTitle,
+              disabled: false, // TODO: add var to set isChatInputSubmitDisabled,
+              iconProps: {
+                iconName: 'Send',
+                styles: inputIconButtonStyles,
+              },
+              onClick: onSubmitInputQuery,
+            }}
+            footerActionsProps={[
+              {
+                title: intlText.actionsButtonTitle,
+                onClick: togglePromptGuide, // TODO: Should open up list of options
+                toggle: true,
+                checked: isPromptGuideOpen,
+              },
+            ]}
+            onQueryChange={(ev, newValue) => {
+              setInputQuery(newValue ?? '');
+            }}
+          ></ChatInput>
+        </div>
+      </div>
     </Panel>
   );
 };
+
+const useStyles = makeStyles({
```

Comment:
We should avoid using component styling - it causes our the CSS to be loaded in runtime, which can cause performance issues: https://github.com/microsoft/fluentui/wiki/Component-Styling